apps/docs/content/components/CTAForm.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: CTAForm
+status: Experimental
+source: https://github.com/primer/brand/tree/main/package/src/CTAForm/CTAForm.tsx
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=1377%3A30754'
+storybook: '/brand/storybook/?path=/story/components-CTAForm--playground'
+description: Use CTAForm to create a form that is used to collect minimal information from users.
+---
+
+import {CTAForm} from '@primer/react-brand'
+
+```js
+import {CTAForm} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTAForm
+  buttonLabel="Subscribe"
+  checkboxLabel={
+    <>
+      I agree to the <a href="www.github.com">terms and conditions</a> and{' '}
+      <a href="www.github.com">privacy policy</a>
+    </>
+  }
+  inputLabel="Your work email address"
+  inputType="email"
+  placeholder="me@github.com"
+/>
+```
+
```

Comment:
Can we have some more examples here please? Like how to conditionally load the checkbox, and remove it from the default state.